server/Entities/Models/Users.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Entities.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPI.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string UserName { get; set; }
+        public string Email { get; set; }
+        public string UserPassword { get; set; }
```

Comment:
just Password

it is property of User entity - we don`t need to specify that it belongs user - it is obvious. we specify belonging if it diffs with entity owner